app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+  end
```

Comment:
Recommend using action filter, and handle in case find_by return nil
```suggestion
class UsersController < ApplicationController
  before_action :load_user, only: :show
  
  def show; end
  
  ...
  private
  ...
  def load_user
    @user = User.find_by id: params[:id]
    return if @user
    
    flash[:error] = ... # Not found message
    redirect_to root_path  
  end
```